### ISSUE TITLE
Completly not working

### ISSUE THREAD

**Issue by Lexumiii** [briansd9/exile-diary#57]
The Programm is doing absolutly nothing since 2 or 3 days.
it doesn't update the stach value or gives me the map info.

**Comment by briansd9** [comment]
Hi, sorry about that, we'll get this fixed.

Can you send me your log file at `%appdata%\exile-diary\log.txt`?

**Comment by Drogzar** [comment]
I have same issue. installed, configured and is doing nothing. Logs seem fine , it gets the character, the league and runs the parser but there is a "No Stash found!" message that keeps popping up, like this:

2020-06-28 14:29:30.245 [info] main.js              > Initializing components
2020-06-28 14:29:30.246 [info] DB.js                > Initializing database for REDACTED
2020-06-28 14:29:30.246 [info] main.js              > Checking current character league...
2020-06-28 14:29:30.247 [info] DB.js                > Database version is 4
2020-06-28 14:29:30.523 [info] main.js              > {"name":"REDACTED","league":"Harvest","classId":3,"ascendancyClass":3,"class":"Necromancer","level":88,"experience":1683735838,"lastActive":true}
2020-06-28 14:29:30.523 [info] ClientTxtWatcher.js  > Watching REDACTED\Path of Exile\logs\Client.txt
2020-06-28 14:29:30.523 [info] InventoryGetter.js   > Inventory getter started with query path /character-window/get-items?league=Harvest&accountName=REDACTED&character=REDACTED
2020-06-28 14:29:30.524 [info] ScreenshotWatcher.js > Watching REDACTED\Documents\My Games\Path of Exile\Screenshots
2020-06-28 14:29:30.524 [info] ItemFilter.js        > Loaded: {"nonunique":{},"unique":{},"gem":{},"map":{},"divcard":{},"prophecy":{},"oil":{},"fragment":{},"delve":{},"catalyst":{},"essence":{},"currency":{}}
2020-06-28 14:29:30.523 [info] main.js              > Done checking, character status is valid
2020-06-28 14:29:30.523 [info] main.js              > Starting components
2020-06-28 14:29:30.528 [info] RateGetterV2.js      > Found existing rates for 20200628
2020-06-28 14:29:32.172 [info] stash.html           > No stash found!

**Comment by briansd9** [comment]
Have you selected which tabs you want to use for calculating net worth? (`Settings -> Stash`) 

If so, it will not update until the threshold you've set there is reached (mine is set to check every 5 maps).

Thanks for reporting this, I will change it to retrieve the stash immediately on first run.

**Comment by Drogzar** [comment]
Yes, I have selected all my tabs and set the threshold to 1/2 hour. It's been1 hour since I posted last comment and done several maps meanwhile, still showing nothing :/

**Comment by briansd9** [comment]
Are your map runs being logged properly? Is the "most recent maps" table on the main page updated with the ones you've done?

Also, can you check your log file for messages from `StashGetter.js`?

**Comment by Drogzar** [comment]
Hmm... no, it WAS not showing anything on the table and the log haD no entry for "StashGetter.js" at all, only the error from stash.html

However, before answering you I decided to restart it again and enter a map and take a screenshot and it immediately showed me the stats + now there are entries of StashGetter.js showing it going through my stash correctly.

Thanks for taking a look and will let you know if it acts up again, but seems like after the first time setup it got in a weird state and restarting it managed to take it out of it. I think your idea of getting the current stash value on start instead of waiting is better usability + should avoid this weird situation.
